ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -48,6 +80,51 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '7881', source: 'Hegemone', capture: false }),
       response: Responses.goSides(),
     },
+    {
+      id: 'P6S Exchange of Agonies Markers',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({}),
+      condition: (data, matches) => {
+        return data.me === matches.target && !(/(00(?:4F|5[0-6]))|013E|0148|0065/).test(getHeadmarkerId(data, matches));
+      },
+      infoText: (data, matches, output) => {
+        const correctedMatch = getHeadmarkerId(data, matches);
+        switch (correctedMatch) {
+          case '0163':
+          case '0167':
+            return output.stackOnYou!();
+          case '0164':
+          case '0165':
+          case '016A':
+            return output.goCorner!();
+          case '0168':
+          case '016E':
+            return output.donut!();
+          default:
+            return output.unknown!();
+        }
+      },
+      outputStrings: {
+        stackOnYou: Outputs.stackOnYou,
+        donut: {
+          en: 'Stack Donut',
+          de: 'Sammeln Donut',
+          fr: 'Packez-vous, donut',
+          ja: '頭割り',
+          cn: '集合放月环',
+          ko: '도넛 장판, 쉐어',
+        },
+        goCorner: {
+          en: 'Go To Corner',
```

Comment:
This is a tiny trigger readability request, but can this be `Spread Corner` or `Spread in Corner` or some such, so that the first word of the callout is either "stack" or "spread"?